helm/templates/deployment.yaml

```diff
@@ -75,14 +75,22 @@ spec:
               containerPort: {{ .Values.containerHealthProbePort }}
               protocol: TCP
           env:
-            - name: SERVICE_NAME
```

Comment:
We may need the service name if we want to override the app name.

Reply:
Responded in first thread, can continue discussion there